02-technical-guides/directquery-vector/main.ipynb

```diff
@@ -5,30 +5,29 @@
    "id": "a6806a84-d79b-4ef5-93bc-8e0f77f3b1c9",
```

Comment:
<p>I can't remember if we decided that we would be adhering to sentence casing moving forward. If this is the case, I can revert this change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Confirmed that we are moving forward with sentence casing, thus I removed this change.